internal/s3assumer/s300006_get_bucket_location_no_such_bucket.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"math/rand"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
```

Comment:
Great to have those here now!

Reply:
@shabbyrobe it just looks like one of the tests is now failing, even though without a lot of information.